lib/SubscriptionScheduleRevision.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Stripe;
+
+/**
+ * Class Subscription
+ *
+ * @property string $id
+ * @property string $object
+ * @property int $created
+ * @property mixed $invoice_settings
+ * @property boolean $livemode
+ * @property mixed $phases
+ * @property string $previous_revision
+ * @property string $renewal_behavior
+ * @property mixed $renewal_interval
+ * @property string $chedule
+ *
+ * @package Stripe
+ */
+class SubscriptionScheduleRevision extends ApiResource
+{
+
+    const OBJECT_NAME = "subscription_schedule_revision";
+
+    use ApiOperations\All;
+    use ApiOperations\Retrieve;
+
+    /**
+     * @return string The API URL for this Subscription Schedule Revision.
+     */
+    public function instanceUrl()
```

Comment:
OOC do we really need this?

Reply:
Not really: `SubscriptionScheduleRevision` doesn't have any instance methods, so this is never called. It's fine to leave it though.